packages/lumx-react/src/components/generic-block/GenericBlock.stories.tsx

```diff
@@ -49,6 +49,22 @@ export const Vertical = ({ theme }: HasTheme) => (
     </GenericBlock>
 );
 
+export const VerticalOverflow = ({ theme }: HasTheme) => (
+    <div style={{ width: 300 }}>
+        <GenericBlock orientation={Orientation.vertical}>
+            <GenericBlock.Figure style={redBorderStyle}>
+                <Icon icon={mdiPencil} size={Size.m} />
+            </GenericBlock.Figure>
+            <GenericBlock.Content style={redBorderStyle}>
+                _________________________________________________________________
+            </GenericBlock.Content>
+            <GenericBlock.Actions fillSpace style={redBorderStyle}>
```

Comment:
Do we want the actions to `fillSpace` by default? It looks like they always are in the stories.

Reply:
No actually here `fillSpace` does not have any effect in vertical mode without height constraint ([see storybook](https://5fbfb1d508c0520021560f10-tcmubuhvry.chromatic.com/?path=/story/lumx-components-generic-block-genericblock--vertical-overflow)) I'll remove it.
It is activated by default on the content section though

Reply:
Ah I see. Anyway, it was nitpicking :) Looks good to me.